packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -154,9 +160,10 @@ export class FacetSearch {
           class="suggestion cursor-pointer flex flex-row items-center px-2 text-sm"
           value={index}
         >
-          <span class="label whitespace-nowrap overflow-ellipsis overflow-hidden">
-            {suggestion.rawValue}
-          </span>
+          <span
+            class="label whitespace-nowrap overflow-ellipsis overflow-hidden"
+            innerHTML={this.highlightSuggestion(suggestion.displayValue)}
```

Comment:
Since the value is now being rendered as HTML, we should escape the original value to prevent XSS attacks.

Reply:
There exist a `sanitize` method in the xss-utils file